src/main/java/ch/njol/skript/classes/Changer.java

```diff
@@ -89,6 +93,29 @@ public static boolean acceptsChange(final Expression<?> e, final ChangeMode mode
 			}
 			return false;
 		}
+
+		/**
+		 * Handles removing string. The 'all' boolean is for if the ChangeMode is REMOVE_ALL
+		 * 
+		 * @param input The String to modify.
+		 * @param toRemove The value to remove from the input string.
+		 * @param all If the ChangeMode is REMOVE_ALL or not
+		 * @return Input formatted with the replacement.
+		 */
+		public static String handleStringRemove(String input, String toRemove, boolean all) {
```

Comment:
This would be better for a general utility class IMO

Reply:
It's under the ChangerUtils class